package.json

```diff
@@ -10,6 +10,7 @@
     "ckeditor5-plugin"
   ],
   "dependencies": {
+    "@ckeditor/ckeditor5-clipboard": "^10.0.3",
```

Comment:
That's not necessary. You're not using it anyway. You're only using `plugins.get( 'Clipboard' )` which can be satisfied by other implementations of this feature (not only `ckeditor5-clipboard`).

Reply:
I see, wouldn't 

```js
this.listenTo( editor.plugins.get( 'Clipboard' ), ... );
```

fail if there will not be any `Clipboard` feature in the CKEditor build?

Reply:
Yes, it would fail. But we do the same thing with other feature-feature relations. This simplifies recomposing the editor with e.g. a bit different implementation of one of those features.

We use `Plugin#requires()` for feature-util and gluefeature-subfeature relations.